### PR TITLE
Milestone 1 | animals table

### PR DESCRIPTION
**Pull request by ITurres**
# 🚩I have made the following changes to complete Vet Clinic - Milestone/Project 1

### Type of **`files`** changed/added

My Skills, **PNG**

### In this new Branch, I have **➕ADDED➕** the succeeding **\*📂 folders** and/or **\*\*📄 files** to complete this milestone's [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

# \*📂 root/

- **\*\*📄 LICENSE** ➕

  - I have added the MIT License to this project.

- ## \*📂 logs/

  - **\*\*📄 animals-query-results.sql** ➕

    - **This [file](https://github.com/ITurres/vet-clinic-db/blob/project-1/animals-table/logs/animals-query-results.sql) also contains the results of the queries made to the database. In addition to the screenshots provided at the end of this PR**

- ## \*📂 src/media/

  - **\*\*📄 vet-clinic-png.png** ➕

### I have **✏️MODIFIED✏️** the succeeding **\*📂 folders** and/or **📄\*\* files**

# \*📂 root/

- **\*\*📄 README.md** ✏️

  - I have modified the documentation following the suggested guidelines.

- **\*\*📄 data.sql** ✏️

  - This file contains the data inserted into the database.

- **\*\*📄 queries.sql** ✏️

  - This file contains the queries made to the database.

- **\*\*📄 schema.sql** ✏️

  - This file contains the database's _`animals`_ table schema.

---

### 📸 Screenshots of queries made to the database

  <details>
    <summary>Click to expand!</summary>

query-1

query-2

query-3

query-4

query-5

query-6

query-7

query-8

  </details>
  
---
---

### Thank you for taking the time to review this PR ⭐

---

If you require additional information or have any questions, please contact me on Slack as _**Arturo (Arthur) Emanuel Guerra Iturres**_. I'll be happy to help you out. 🎯

---

### CODE REVIEW

File: logs/animals-query-results.sql
Comment (LionRouge1) [inline logs/animals-query-results.sql:1]: - [x] Great work adding the queries results :clap: However it requested in the project requirement  to take a screenshot of the results of each queries and add them to your Pull Request description(see image). Kindly add the screenshot of your querie's result to your PR to follow the project requirement.
Screenshot from 2023-08-15 00-59-24

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @ITurres, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Use correct `sql` commands :heavy_check_mark: 
- Project generated using [this template](https://github.com/microverseinc/curriculum-template-databases) :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: logs/animals-query-results.sql
Comment (ITurres) [reply logs/animals-query-results.sql:1]: Hi @LionRouge1! great to see you again! 😃, hope you are well.

Thank you, Lion! Yes! I completely forgot to add the image files. I will do so right away 🎯. 

Cheers, and happy coding partner!

**Review by Davidosky007 — APPROVED** [review]
Hi @ITurres 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Insert and query data in SQL properly 👍 
* Well written PR 👍 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ITurres** [comment]
Hi @Davidosky007 David! Good to see you too 😃👋. Cheers!

**Pull request merged**